### DISCUSSION TITLE
Properties with internal scope

### DISCUSSION THREAD

**Discussion by sigmarsson** [canton7/Stylet#279] [Q&A]
Would it be possible to access `internal` properties as well ? Currently only `public` ones are visible on the `View-Model`.

**Comment by canton7** [comment]
Access in what sense? Visible to whom?

**Reply by sigmarsson** [reply]
Visible to the View. If the props are internal, the bindings aren't accomplished. Maybe this is related how the `Binding` works and reads only public ones ? I am not sure, perhaps `x:Bind` in WinUI can cope with the internal ones as well. But I said, I am not sure, maybe this is how `Binding` works and unrelated to Stylet.  I witness the same as for the guard props of a `Button` for ex.

**Reply by canton7** [reply]
Yes, this is an issue about `Binding` -- Stylet isn't involved with bindings at all

**Reply by sigmarsson** [reply]
`Yeah, Thank you!` It is just vexing because it hence enforces to alter scopes of other types as well.
